Apps/Database/src/Models/MessagingVerification.cs

```diff
@@ -47,7 +47,7 @@ public class MessagingVerification : AuditableEntity
         /// Gets or sets the associated email that was sent for this verification.
         /// Required if the VerificationType = MessagingVerificationType.Email.
         /// </summary>
-        public Guid EmailId { get; set; }
+        public Guid? EmailId { get; set; }
```

Comment:
This seems to have changed the column type... Do we want that?

Reply:
Yes, emailId is nullable now.